app/test/dartdoc/golden/pana_0.10.2_license_file_name_field.html

```diff
@@ -53,13 +53,13 @@ <h5>LicenseFile class</h5>
       <li><a href="pana/LicenseFile/shortFormatted.html">shortFormatted</a></li>
       <li><a href="pana/LicenseFile/url.html">url</a></li>
       <li><a href="pana/LicenseFile/version.html">version</a></li>
-      <li class="inherited"><a href="https://api.dartlang.org/dev/2.0.0-dev.59.0/dart-core/Object/runtimeType.html">runtimeType</a></li>
+      <li class="inherited"><a href="pana/LicenseFile/runtimeType.html">runtimeType</a></li>
     
       <li class="section-title"><a href="pana/LicenseFile-class.html#instance-methods">Methods</a></li>
       <li><a href="pana/LicenseFile/change.html">change</a></li>
       <li><a href="pana/LicenseFile/toString.html">toString</a></li>
-      <li class="inherited"><a href="https://api.dartlang.org/dev/2.0.0-dev.59.0/dart-core/Object/noSuchMethod.html">noSuchMethod</a></li>
-      <li class="inherited">toJson</li>
+      <li class="inherited"><a href="pana/LicenseFile/noSuchMethod.html">noSuchMethod</a></li>
```

Comment:
These seem strictly worse...

Reply:
Why is that? (Btw. not sure how we could remove that part easily, as it is coming from `dartdoc` and is affecting all of the higher-level pages too.)

Reply:
https://api.dartlang.org/dev/2.0.0-dev.59.0/dart-core/Object/noSuchMethod.html seems valid

pana/LicenseFile/noSuchMethod.html can't imagine this is correct

Is this a bug w/ dartdoc? CC @jcollins-g

Reply:
This looks like something has happened to where dartdoc no longer is linking externally.  That's a bug, but I don't know if it is something having to do with the environment this is run in or native to dartdoc.  Need more information (including stdout from dartdoc and exactly how it is being run to generate this).

Reply:
I believe this is run with `--input <dir> --output <dir> --hosted-url <url> --rel-canonical-prefix <url> --link-to-remote`.